### PR TITLE
refactor(vtkmath): remove interdependencies (vtkMath and vtkMatrixBuilder)

### PR DESCRIPTION
**Pull request by laurennlam**
@finetjul @jourdain

### CODE REVIEW

File: Sources/Interaction/Manipulators/MouseCameraAxisRotateManipulator/index.js [hunk 1]
```diff
@@ -128,7 +129,12 @@ function vtkMouseCameraAxisRotateManipulator(publicAPI, model) {
         const fpDist = vec3.length(newFp);
 
         // Note it normalizes the vector to be rotated
-        const result = vtkMath.rotateVector(newFp, v2, rAngle);
+        const result = [...newFp];
+        vtkMatrixBuilder
+          .buildFromRadian()
+          .identity()
```
Comment (finetjul) [inline Sources/Interaction/Manipulators/MouseCameraAxisRotateManipulator/index.js:135]: couldn't you get rid of identity matrix ?

Comment (laurennlam) [reply Sources/Interaction/Manipulators/MouseCameraAxisRotateManipulator/index.js:135]: Indeed. As the constructor already set the matrix to the identity.

File: Sources/Widgets/Widgets3D/ResliceCursorWidget/helpers.js [hunk 2]
```diff
@@ -149,6 +151,16 @@ export function getViewPlaneNameFromViewType(viewType) {
   }
 }
 
+export function rotateVector(vectorToBeRotated, axis, angle) {
```
Comment (finetjul) [inline Sources/Widgets/Widgets3D/ResliceCursorWidget/helpers.js:154]: your test could be applied for this function

File: Sources/Widgets/Widgets3D/ResliceCursorWidget/test/testRotateVector.js [hunk 3]
```diff
@@ -1,12 +1,10 @@
...
-  radiansFromDegrees,
-  areEquals,
-} from 'vtk.js/Sources/Common/Core/Math';
+import { radiansFromDegrees, areEquals } from 'vtk.js/Sources/Common/Core/Math';
 
-test('Test vtkMath rotateVector', (t) => {
+import rotateVector from 'vtk.js/Sources/Widgets/Widgets3D/ResliceCursorWidget/helpers';
+
+test('Test vtkResliceCursorHelper rotateVector', (t) => {
   function degreesToRadians(degrees) {
```
Comment (finetjul) [inline Sources/Widgets/Widgets3D/ResliceCursorWidget/test/testRotateVector.js:8]: does not seem useful

**Comment by finetjul** [comment]
LGTM

**Pull request merged**